app/publishing_api_docs.rb

```diff
@@ -21,12 +21,24 @@ def initialize(filename, title)
       @title = title
     end
 
+    def branch
+      'master'
```

Comment:
I think we can make `master` the only option, maybe we shouldn't allow other branches.